migrations/versions/110_add_acknowledged_column.py

```diff
@@ -0,0 +1,41 @@
+"""empty message
+
+Revision ID: 2d2309ce61f
```

Comment:
The revisions/revises in the comments in these migration scripts are all wrong.  Would be nice if they match up to the revision/down_revision numbers, for clarity/sanity.